applications/content-creator-applications/domain/contentCreatorApplication.js

```diff
@@ -1,38 +1,35 @@
 module.exports = function buildMakeContentCreatorApplication({ Id }) {
 
-	return function makeContentCreatorApplication({
-		id = Id.makeId(),
-		firstName,
-		lastName,
-		email,
-		motivation,
-		approved = false,
-		createdAt = new Date(),
-		modifiedAt = new Date()
-	}) {
+    return function makeContentCreatorApplication({
+        id = Id.makeId(),
+        name,
+        email,
+        password,
+        approved = false,
+        createdAt = new Date(),
+        modifiedAt = new Date()
+    }) {
 
-		if (!firstName) throw new Error('A firstname must be provided in the application');
-		if (!lastName) throw new Error('A lastname must be provided in the application');
-		if (!email) throw new Error('An email must be provided in the application');
+        if (!name) throw new Error('A name must be provided in the application')
+        if (!email) throw new Error('An email must be provided in the application')
 
-		let rejectReason = '';
+        let rejectReason = ''
 
-		return Object.freeze({
-			getId: () => id,
-			getFirstName: () => firstName,
-			getLastName: () => lastName,
-			getEmail: () => email,
-			getMotivation: () => motivation,
-			isApproved: () => approved,
-			getRejectReason: () => rejectReason,
-			getCreatedAt: () => createdAt,
-			getModifiedAt: () => modifiedAt,
-			fullname: () => `${firstName} ${lastName}`,
-			approve: () => approved = true,
-			decline: ({ reason = 'No reason given' } = {}) => {
-				approved = false;
-				rejectReason = reason;
-			}
-		});
-	};
-};
\ No newline at end of file
+        return Object.freeze({
+            getId: () => id,
+            getname: () => name,
```

Comment:
for now the code would not work if we put in first name and last name as its currently setup. as explained, this will be changed in a future sprint, but for now we'll need to keep it to name